snippets/dune-project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "lang": {
+    "prefix": "lang",
+    "body": ["(lang $1 $2)"]
+  },
+  "name": {
+    "prefix": "name",
+    "body": ["(name $1)"]
+  },
+  "using": {
+    "prefix": "using",
+    "body": ["(using $1 $2)"]
+  },
+  "info": {
+    "prefix": "info",
+    "body": [
+      "(license $1)\n(authors \"$2\")\n(maintainers \"$3\")\n(source (github $4))\n(bug_reports \"$5\")\n(homepage \"$6\")\n(documentation \"$7\")"
```

Comment:
I don't know how prevalent github is in the ocaml community, but users aren't necessarily going to be using it.
Maybe this should default to `uri` instead of `github`.

Reply:
Indeed! I made it possible to choose. What do you think?
![image](https://user-images.githubusercontent.com/38746192/80871653-a1054580-8ce8-11ea-9cd4-e114525b71f4.png)

Reply:
Looks good